lib/local_system.rb

```diff
@@ -67,6 +67,16 @@ def validate_existence_of_packages(packages)
       end
     end
 
+    def validate_existence_of_command(command, package)
+      Cheetah.run("which", command)
```

Comment:
The problem is that `which` is not always installed, at least it could be since it is part of the "which" package in openSUSE.